platform/windows/display_server_windows.cpp

```diff
@@ -2736,6 +2736,53 @@ void DisplayServerWindows::window_request_attention(WindowID p_window) {
 	FlashWindowEx(&info);
 }
 
+void DisplayServerWindows::window_set_taskbar_progress_value(int p_value, int p_total, WindowID p_window) {
+	_THREAD_SAFE_METHOD_
+
+	ERR_FAIL_COND(!windows.has(p_window));
+	const WindowData &wd = windows[p_window];
+
+	CoInitialize(0);
+	CoCreateInstance(CLSID_TaskbarList, 0, CLSCTX_INPROC_SERVER, IID_ITaskbarList, (void **)&pTaskbar);
+	pTaskbar->HrInit();
+
+	pTaskbar->SetProgressValue(wd.hWnd, (ULONGLONG)p_value, (ULONGLONG)p_total);
+}
+
+void DisplayServerWindows::window_set_taskbar_progress_state(ProgressState p_state, WindowID p_window) {
+	_THREAD_SAFE_METHOD_
+
+	ERR_FAIL_COND(!windows.has(p_window));
+	const WindowData &wd = windows[p_window];
+
+	CoInitialize(0);
```

Comment:
`os_windows.cpp` already calls `CoInitializeEx(nullptr, COINIT_APARTMENTTHREADED);` so I don't think you need to call `CoInitialize(0)` here.
Same above.